### PR TITLE
feat: First loading speed improvement

### PR DESCRIPTION
**Pull request by Razzwan**
resolves: https://github.com/daostack/alchemy/issues/2260

### CODE REVIEW

**Comment by orenyodfat** [comment]
is there a way to measure loading speed so we can track the improvement ?

**Comment by Razzwan** [comment]
> is there a way to measure loading speed so we can track the improvement ? @Razzwan

I can just measure loading speed on my compute before and after. Will do that

**Comment by Razzwan** [comment]
## Before
Снимок экрана 2020-11-17 в 12 29 25

## After
Снимок экрана 2020-11-17 в 12 17 41

Also, please take a look [this video](https://www.loom.com/share/c79f2aecbc8a4aa8b4ef1c5cbc5015fb)

**Review by roienatan — COMMENTED** [review]
Why some files have been renamed?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## 1.0.2
   - Features Added
+    - First loading speed improvement
```
Comment (roienatan) [inline CHANGELOG.md:3]: Please remove it from here, 1.0.2 has already been released. This fix will go to the next release.

**Review by roienatan — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Razzwan) [reply CHANGELOG.md:3]: Fixed. Added new section

Comment (Razzwan) [reply CHANGELOG.md:3]: @roienatan anything else?

**Comment by Razzwan** [comment]
> Why some files have been renamed?

Which exactly?
1. `test/integration/wdio.conf.js` -> `test/integration/wdio.conf.ts` to add typescript supporting. It helps to avoid configurational problesm when packages are updated
2. `test/integration/*.ts` -> `test/integration/*.e2e.ts` to be able correctly exclude files from test scope (such as `utils.ts` and `test/integration/wdio.conf.ts`) from test scope

File: CHANGELOG.md (hunk 1 above)
Comment (roienatan) [reply CHANGELOG.md:3]: Let's wait for @orenyodfat review.

**Review by orenyodfat — APPROVED** [review]

**Pull request merged**